zkstack_cli/crates/zkstack/build.rs

```diff
@@ -1,13 +1,18 @@
 use std::path::PathBuf;
 
+use dirs::{config_local_dir, home_dir};
 use ethers::contract::Abigen;
+use eyre::ContextCompat;
```

Comment:
Could you try to use anyhow? 
seems it's the only place where we use eyre

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3097/commits/ce756763c84221a3433800f56ef3c63ab5f96ffa